### PR TITLE
FUSETOOLS-2171 - upgrade to Neon TP 4.4.0.CR1

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -41,7 +41,8 @@
     <maven.compiler.target>1.8</maven.compiler.target>
 
     <!-- JBoss Tools Integration Stack version -->
-    <jbtis.version>4.4.0.CR1-SNAPSHOT</jbtis.version>
+    <jbtis.version>4.4.0.CR1</jbtis.version>
+    <jbtis.classifier>base</jbtis.classifier> <!-- base-ea -->
```
Comment (apupier) [inline pom.xml]: why the classifier has changed from base-ea to base ?

Comment (lhein) [inline pom.xml]: I can ask Paul later but thats what he told me to change.

Comment (apupier) [inline pom.xml]: ok

**Comment by apupier** [comment]
do we need to align the jboss-tools-parent version?
we are at 4.4.0.Alpha1-SNAPSHOT
on http://repository.jboss.org/nexus/content/repositories/public/org/jboss/tools/parent/
there are 4.4.0.Final-SNAPSHOT, 4.4.1.Alpha1-SNAPSHOT, 4.4.1.Final-SNAPSHOT, 4.4.2.AM1-SNAPSHOT

maybe 4.4.1.Final-SNAPSHOT (or even 4.4.1.Final if we can find it somewhere)

**Comment by apupier** [comment]
testPR

**Comment by lhein** [comment]
I see no urgent requirement to do so

**Pull request merged**